pkg/bootstrap/docker/openshift/logging.go

```diff
@@ -50,6 +51,47 @@ func instantiateTemplate(client client.Interface, mapper configcmd.Mapper, templ
 	return nil
 }
 
+// InstallLoggingViaAnsible checks whether logging is installed and installs it if not already installed
+func (h *Helper) InstallLoggingViaAnsible(f *clientcmd.Factory, serverIP, publicHostname, loggerHost, imagePrefix, imageVersion, hostConfigDir, imageStreams string) error {
+	_, kubeClient, err := f.Clients()
+	if err != nil {
+		return errors.NewError("cannot obtain API clients").WithCause(err).WithDetails(h.OriginLog())
+	}
+
+	_, err = kubeClient.Core().Namespaces().Get(loggingNamespace, metav1.GetOptions{})
+	if err == nil {
+		// If there's no error, the logging namespace already exists and we won't initialize it
+		return nil
+	}
+
+	// Create logging namespace
+	out := &bytes.Buffer{}
+	err = CreateProject(f, loggingNamespace, "", "", "oc", out)
+	if err != nil {
+		return errors.NewError("cannot create logging project").WithCause(err).WithDetails(out.String())
+	}
+
+	params := newAnsibleInventoryParams()
+	params.Template = defaultLoggingInventory
+	params.MasterIP = serverIP
+	params.MasterPublicURL = fmt.Sprintf("https://%s:8443", publicHostname)
+	params.OSERelease = imageVersion
+	params.LoggingImagePrefix = fmt.Sprintf("%s-", imagePrefix)
+	params.LoggingImageVersion = imageVersion
+	params.LoggingNamespace = loggingNamespace
+	params.KibanaHostName = loggerHost
+
+	runner := newAnsibleRunner(h, kubeClient, loggingNamespace, imageStreams, "logging")
+
+	//run logging playbook
+	_, err = runner.RunPlaybook(params, loggingPlaybook, hostConfigDir, imagePrefix, imageVersion)
```

Comment:
It doesn't seem that RunPlaybook returns any string but maybe you meant to use it in some other way?